modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/parser/java/JavaExtensionModelParserTestCase.java

```diff
@@ -44,6 +47,15 @@ public class JavaExtensionModelParserTestCase {
   @Rule
   public ExpectedException expectedException = ExpectedException.none();
 
+  @Test
+  @Description("Verify that an extension with Jdk field with encapsulated internal fields does not fail due to reflection in Java 17")
```

Comment:
`an extension with Jdk field with encapsulated internal fields` -> what does this phrase mean? Please rephrase.